src/intersection-observer-polyfill.js

```diff
@@ -86,7 +86,7 @@ export class IntersectionObserverApi {
    * @param {!Element} iframe
    * @param {boolean=} opt_is3p
    */
-  constructor(baseElement, iframe, opt_is3p) {
+  constructor(baseElement, iframe, sentinel, opt_is3p) {
```

Comment:
you might need to change `intersection-observer.js` as well now. it's unfortunately reverted from trash.